backend/src/main/java/ai/verta/modeldb/metadata/MetadataDAORdbImpl.java

```diff
@@ -185,11 +211,13 @@ public boolean deleteLabels(IdentificationType id, List<String> labels, boolean
             .append(ModelDBConstants.ID)
```

Comment:
deleteLabels(
      Session session, IdentificationType id, List<String> labels, boolean deleteAll) seem to be not used anymore too

Reply:
Method removed